docs/content/modules/player-lifecycle-feature-module.md

```diff
@@ -14,49 +15,53 @@ Before reading this document, make sure you are familiar with:
 After your worker has been created, you need to use the following code snippet:
 
 1. On a [client-worker]({{urlRoot}}/content/glossary#client-worker): `PlayerLifecycleHelper.AddClientSystems(Worker.World)`
+  * By default, the module automatically sends a player creation request as soon as the client worker connects to SpatialOS. To manually initiate player creation or provide arbitrary data in a player creation request, add an extra `false` argument when adding client systems: `PlayerLifecycleHelper.AddClientSystems(Worker.World, false);`
 1. On a [server-worker]({{urlRoot}}/content/glossary#server-worker): `PlayerLifecycleHelper.AddServerSystems(Worker.World)`
 
-### How to spawn a player entity
+## How to spawn a player entity
 
 To spawn a [SpatialOS entity]({{urlRoot}}/content/glossary#spatialos-entity) representing
 a player, you need to:
 
 1. Define the entity template for your player entity
 1. Specify an entity template for player creation
+1. (Optional) Manually request player creation
+
+>**NOTE: To pass in arbitrary serialized data with the player creation request, `AutoRequestPlayerCreation` must be set to false.**
 
-The client-worker initiates the player entity creation once it has been successfully created.
+If `AutoRequestPlayerCreation` is set to false you must manually initiate player creation, as described in Step 3.
 
-#### 1. Define the player entity template
+##### 1. Define the player entity template
 
 The module takes care of spawning the player entity as soon as a client-worker connects. To enable this behaviour, the server-worker responsible for handling requests to spawn player entities needs to know which [entity template]({{urlRoot}}/content/entity-templates) to use when sending the entity creation request to the [SpatialOS Runtime]({{urlRoot}}/content/glossary#spatialos-runtime).
 
 Create a method that returns an `EntityTemplate` object and takes the following parameters to define your player [entity template]({{urlRoot}}/content/entity-templates):
 
-* `string workerId`: The ID of the worker that wants to spawn this player entity
-*  `Improbable.Vector3f position`: the position where  the client-worker requested to spawn the player entity
+* `string workerId`: The ID of the worker that wants to spawn this player entity.
+*  `byte[] playerCreationArguments`: a serialized byte array of arguments provided by the worker sending the player creation request.
 
-When defining the entity template, you need to use the `AddPlayerLifecycleComponents` method.
-This methods adds the SpatialOS components to the player entity template necessary to manage the player lifecycle for this entity.
+When defining the entity template, you need to use the `AddPlayerLifecycleComponents` method. This methods adds the SpatialOS components to the player entity template necessary to manage the player lifecycle for this entity.
```

Comment:
```suggestion
When defining the entity template, you need to use the `AddPlayerLifecycleComponents` method. This method adds lifecycle SpatialOS components to the player entity template; these components are necessary to manage the player lifecycle for this entity.
```
Just tried to make the last sentence flow without stumbling - original took a few reads to follow.